CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ### Master
 
+#### Emission
```

Comment:
note the switch to 4 `#`s - by doing this I can put the CHANGELOG as splitting on the versions